dataactcore/scripts/loadExecComp.py

```diff
@@ -0,0 +1,176 @@
+import logging
+import os
+import sys
+import pandas as pd
+import paramiko
+import zipfile
+import re
+from collections import OrderedDict
+import numpy as np
+import argparse
+
+from dataactcore.models.domainModels import ExecutiveCompensation
+from dataactcore.interfaces.db import GlobalDB
+from dataactcore.logging import configure_logging
+from dataactvalidator.health_check import create_app
+from dataactvalidator.scripts.loaderUtils import insert_dataframe
+from dataactcore.config import CONFIG_BROKER
+
+
+logger = logging.getLogger(__name__)
+
+REMOTE_SAM_DIR = '/current/SAM/6_EXECCOMP'
+# sftp -i SAMDATAProd -P 22 samdataprod04@66.77.18.174
```

Comment:
Should this comment still be here? Seems like something we probably don't want just hanging around. If it's ok then I don't mind, just wanted to make sure we weren't putting important information in the code.